### PR TITLE
Components

### PR DESCRIPTION
**Pull request by rojintag**

### CODE REVIEW

File: tie-in-frontend/src/components/ApplicationCard.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  return (
+    <div className="application-card">
+      <h3>Name</h3>
+      <p>{companyProjectName}</p>
+      <h3>Applicant</h3>
+      <p>{name}</p>
+      <img src={image} alt="student's image" />
+      <h3>Status</h3>
+      <p>{status}</p>
+
```
Comment (FloridaLangara) [inline tie-in-frontend/src/components/ApplicationCard.js:15]: Remove extra spaces

File: tie-in-frontend/src/components/BusinessProjectDetails.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      <p>{teamRequirement}</p>
+      <p>Project start date:</p>
+      <p>{startDate}</p>
+      <p>Project end date:</p>
+      <p>{endDate}</p>
+      <p>Estimated Budget:</p>
+      <p>{budget}</p>
+      <p>Location:</p>
+      <p>{projectLocation}</p>
+
```
Comment (FloridaLangara) [inline tie-in-frontend/src/components/BusinessProjectDetails.js:24]: same here

**Review by FloridaLangara — COMMENTED** [review]
Please fix the typos and structure your code. Be careful with additional spaces.

File: tie-in-frontend/src/components/ApplicationCard.js (hunk 1 above)
Comment (rojintag) [reply tie-in-frontend/src/components/ApplicationCard.js:15]: Done

**Pull request merged**